### PR TITLE
Binary classification with SymbolicRegression

### PR DESCRIPTION
**Pull request by guyko81**
The current SymbolicRegressor class is used with different metric. The metric contains the sigmoid function for transforming the output of the model. 
TODO: The output of the model should be transformed if one wants to use it. It generates the raw logit values now.

### CODE REVIEW

**Comment by trevorstephens** [comment]
Hey @guyko81 ... I'm not sure what this PR is about. As I mentioned in #24 work is already underway to create a classifier in #141 ... Is there something there that will not work for you? I welcome any reviews on that PR for implementation/usefulness/etc! Due to the scikit-learn API I would not want to implement a classifier within the regressor (though I'm sure plenty of people have hacked it that way to get it to work before the next release haha), it would not be able to pass tests.

**Comment by guyko81** [comment]
Hi @trevorstephens, I just wanted to share the idea and my solution. If you can use any part then I'm happy. Or if you throw away it's OK too, I guess you also have your own idea. 
So the only purpose of this pull request was to show my solution in code.

**Comment by trevorstephens** [comment]
Fair enough @guyko81 :-) We have very similar ideas on the solution! Hope that your needs are met by this

**Comment by trevorstephens** [comment]
Closing in favour of #141
